modules/form/src/components/adapters.tsx

```diff
@@ -20,7 +20,8 @@ import { Button, CopyInputField, DangerButton, LinkButton, Password, PrimaryButt
 import omit from "lodash-es/omit";
```

Comment:
Add missing license header

Reply:
The header already exists. Hence resolving this.